pinot-controller/src/main/java/org/apache/pinot/controller/helix/ControllerRequestURLBuilder.java

```diff
@@ -211,6 +211,9 @@ public String forTableView(String tableName, String view, @Nullable String table
     }
     return url;
   }
+  public String forTableSchemaGet(String tableName) {
+    return StringUtil.join("/", _baseUrl, "tables", tableName, "schema");
```

Comment:
Better to use `File.separator` than `/`?

Reply:
I feel it should be `/` since it's for constructing URL, not filesystem path.